x/cdp/abci.go

```diff
@@ -36,6 +39,14 @@ func BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock, k keeper.Keeper)
 			panic(err)
 		}
 
+		if blockHeight%params.BeginBlockerExecutionBlockInterval != 0 {
+			ctx.Logger().Debug("skipping x/cdp SynchronizeInterestForRiskyCDPs and LiquidateCdps")
+			return
+		}
+
+		err = k.LiquidateCdps(ctx, cp.LiquidationMarketID, cp.Type, cp.LiquidationRatio, cp.CheckCollateralizationIndexCount)
```

Comment:
Looks like a copypasta mistake -- this LiquidateCdps shouldn't be added as it is run below